<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddToPassengerTable</class>
 <widget class="QWidget" name="AddToPassengerTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AirPortSystems:Add Passenger</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="SortPlane">
     <property name="text">
      <string>Сортировать</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTableView" name="PlanesTable">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="RegistrationPassenger">
     <property name="text">
      <string>Зарегистровать пассажира</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignLeft">
    <widget class="QComboBox" name="PlaneSortBox"/>
   </item>
   <item row="0" column="2" alignment="Qt::AlignRight">
    <widget class="QTableView" name="UserTable">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="exit">
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="SortUser">
     <property name="text">
      <string>Найти По Имени и Фамилии</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
